src/postgres/format_tables.sql

```diff
@@ -3,6 +3,8 @@ ALTER TABLE titles
     USING
         string_to_array(genres, ',')::varchar(20)[];
 
+ALTER TABLE ratings
+    ALTER COLUMN averageRating TYPE numeric(2,1);
```

Comment:
We should just change the type in `create_tables.sql`. No point in loading it in `double precision` first then changing the type.

Reply:
I just pushed a new commit as per your request to change the type in `create_tables.sql` file instead of `format_tables.sql`